src/Elcodi/Bundle/AttributeBundle/CompilerPass/MappingCompilerPass.php

```diff
@@ -26,11 +26,7 @@
 class MappingCompilerPass extends AbstractMappingCompilerPass
 {
     /**
-     * You can modify the container here before it is dumped to PHP code.
```

Comment:
Because it was just copied from https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/DependencyInjection/Compiler/CompilerPassInterface.php#L26 and here it doesn’t represent any meaningful information. Should I readd it?